### PR TITLE
committing FeatureA

### PR DESCRIPTION
**Pull request by gururaj3091**
Hi
I have completed the new Feature, pls review it

### CODE REVIEW

File: src/com/exp/sample/SampleFeature.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.exp.sample;
+
+public class SampleFeature {
+
+	public static void main(String[] args) {
+		System.out.println("Feature added by Engineer");
+
+	}
+
```
Comment (gururajanayakr) [inline src/com/exp/sample/SampleFeature.java:11]: write the feature name in the program

**Review by gururajanayakr — COMMENTED** [review]
Please change the statements

**Comment by gururaj3091** [comment]
I have Added the condition and the feature name

**Comment by gururajanayakr** [comment]
code Looks good

**Pull request closed without merging**